docs/zq/functions/cast.md

```diff
@@ -0,0 +1,60 @@
+### Function
+
+&emsp; **cast** &mdash; coerce a value to a different type
+
+### Synopsis
+
+```
+cast(val: any, t: type) -> any
+```
+
+### Description
+
+The _cast_ function performs type casts but handles both primitive types and
+complex types.  If the input type `t` is a primitive type, then the result
+is equivalent to
+```
+<t>(<val>)
+```
+e.g., the result of `cast(1, <string>)` is the same as `string(1)` which is `"1"`.
```

Comment:
It's also equivalent to <string>(1) (you can also use complex types for this syntax).

Reply:
This is explained in language.md and I don't feel like it needs to be included here too as most the of the time you wouldn't type the angle brackets for primitive types.

Reply:
I updated language.md with a bit more clarity on this...